code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -149,8 +149,10 @@
 			user.visible_message("[user] places [src] against [user.p_their()] chest and listens attentively.", "You place [src] against your chest...")
 		else
 			user.visible_message("[user] places \the [src] against [M]'s chest and listens attentively.", "You place \the [src] against [M]'s chest...")
-		var/obj/item/organ/internal/H = M.get_int_organ(/obj/item/organ/internal/heart)
-		var/obj/item/organ/internal/L = M.get_int_organ(/obj/item/organ/internal/lungs)
+		var/datum/organ/heart/heurt = M.get_int_organ_datum(ORGAN_DATUM_HEART)
+		var/obj/item/organ/internal/H = heurt.linked_organ
+		var/datum/organ/lungs/loongs = M.get_int_organ_datum(ORGAN_DATUM_LUNGS)
+		var/obj/item/organ/internal/L = loongs.linked_organ
```

Comment:
whats with the names of these variables

Reply:
They funny tho

Loooooongs

Reply:
> whats with the names of these variables

i ran out of variable names please ignore any variables named snorgan, borgan, norgan, or deadgan.
(I can change them back if needed, but it makes it easier to read shit sometimes, even if theyre a bit silly)